dbase.py

```diff
@@ -0,0 +1,19 @@
+import mysql.Connector

```

Comment:
All these are not actually needed, we don't need to write our own connectors

Reply:
Okay